js/service.js

```diff
@@ -1,11 +1,11 @@
 import env from "../config/env.generated.json";
+import BluebirdPromise from "bluebird";
```

Comment:
let me try and see if i can write something myself instead of using `bluebird` just for this.